src/Client.php

```diff
@@ -355,8 +355,8 @@ public function __get($name)
     }
 
     /**
-     * @param $name
-     * @param $value
+     * @param        $name
+     * @param        $value
```

Comment:
Let's use the type that the php documentation suggests here.

Reply:
Done